unattended_scripts/install_functions/opendistro/common.sh

```diff
@@ -67,13 +67,13 @@ installPrerequisites() {
     logger "Installing all necessary utilities for the installation."
 
     if [ ${sys_type} == "yum" ]; then
-        eval "yum install curl unzip wget libcap tar -y ${debug}"
+        eval "yum install curl unzip wget libcap tar openssl -y ${debug}"
     elif [ ${sys_type} == "zypper" ]; then
-        eval "zypper -n install curl unzip wget ${debug}"         
+        eval "zypper -n install curl unzip wget openssl ${debug}"
         eval "zypper -n install libcap-progs tar ${debug} || zypper -n install libcap2 tar ${debug}"
     elif [ ${sys_type} == "apt-get" ]; then
         eval "apt-get update -q $debug"
-        eval "apt-get install apt-transport-https curl unzip wget libcap2-bin tar -y ${debug}"
+        eval "apt-get install apt-transport-https curl unzip wget libcap2-bin tar openssl -y ${debug}"
```

Comment:
Forcing the installation of openssl may not be a good idea, it is possible that the user has a certain and lower version than the latest one. This should be a check in wazuh-cert-tool.sh that checks if there is any version of openssl on the system.

Even better, double check, both in wazuh-cert-tool.sh and in wazuh_install.sh so that the installation does not start if the system do not have it to create the certificates.

And if the user has their own certificates and does not generate them but imports them? Can an unattended install work without openssl on the system?

Reply:
Fixed in https://github.com/wazuh/wazuh-packages/pull/1110/commits/f75864a5d1031e127f0fd67a243eab6d0f5f3c3d and https://github.com/wazuh/wazuh-packages/pull/1110/commits/800745a0533ffedba960200e7125a3130059360b. The comprobation in wazuh_install will be added in https://github.com/wazuh/wazuh-packages/pull/1098

Reply:
Good approximation, please, check the new review